packages/odyssey-storybook/src/components/odyssey-mui/Banner/Banner.mdx

```diff
@@ -68,10 +68,6 @@ Banner content should be succinct and direct. If possible, your content should b
 
 When including an action, be sure the link text clearly indicates where it leads.
 
-## Props
-
-<ArgsTable />
-
```

Comment:
Did you mean to remove this?

Reply:
Yup; in our current setup, these don't display anything. We had removed it from most files, but this one was lingering.